From cb55ed7d958cf4abb58dd1d6e46e09447b5694b0 Mon Sep 17 00:00:00 2001
From: Dmitri Vorobiev <dmitri.vorobiev@gmail.com>
Date: Mon, 12 May 2008 17:49:19 +0400
Subject: [PATCH 1/1] [MIPS] Export empty_zero_page as a GPL symbol

The empty_zero_page symbol is needed for the ext4 driver and
should therefore be exported. This fixes the following error
reported by Martin Michlmayr:

>>>>>>>

MODPOST 1516 modules
ERROR: "empty_zero_page" [fs/ext4/ext4dev.ko] undefined!

>>>>>>

Signed-off-by: Dmitri Vorobiev <dmitri.vorobiev@gmail.com>
---
 arch/mips/mm/init.c |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/arch/mips/mm/init.c b/arch/mips/mm/init.c
index ecd562d..618a418 100644
--- a/arch/mips/mm/init.c
+++ b/arch/mips/mm/init.c
@@ -70,7 +70,10 @@ DEFINE_PER_CPU(struct mmu_gather, mmu_gathers);
  * any price.  Since page is never written to after the initialization we
  * don't have to care about aliases on other CPUs.
  */
-unsigned long empty_zero_page, zero_page_mask;
+unsigned long empty_zero_page;
+EXPORT_SYMBOL_GPL(empty_zero_page);
+
+unsigned long zero_page_mask;
 
 /*
  * Not static inline because used by IP27 special magic initialization code
-- 
1.5.3

